docs/conf.py

```diff
@@ -48,6 +48,9 @@
 
 autoapi_dirs = ['../notfound']
 autoapi_add_toctree_entry = False
+def setup(app):
+  app.add_description_unit('confval', 'confval',
```

Comment:
`.add_description_unit` is deprecated and removed in Sphinx 2.0. We should use `.add_object_type`. See the docs at https://www.sphinx-doc.org/en/1.8/extdev/appapi.html#sphinx.application.Sphinx.add_object_type

It seems it works in the same way.

Reply:
Done.